ironfish/src/rpc/routes/wallet/utils.ts

```diff
@@ -101,6 +101,28 @@ export async function serializeRpcWalletTransaction(
   }
 }
 
+export const serializeRpcImportAccount = (encoded: AccountImport): RpcAccountImport => {
```

Comment:
Maybe name this `accountImport` instead of `encoded`?

Reply:
done